Tasks/Common/azure-blobstorage-artifactProvider-v2/azureBlobStorageProvider.ts

```diff
@@ -0,0 +1,165 @@
+import path = require('path');
```

Comment:
Why create a new version of azure-blobstorage-artifactProvider? Does task lib introduce a breaking change in its behavior?

Reply:
reasoning same as above.

Reply:
change in task lib does not necessarily mean new version of common libraries. Is the functionality being provided by the common lib change due to task lib version update?
By this logic, even task version should get major version update.